README.md

```diff
@@ -1,5 +1,97 @@
+# Awesome Semantic Shapes [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
 
-# Specification 'awesome-semantic-shapes'
+> A curated list of Semantic Shapes resources. Contributions welcome! Please, read the [Contribution Guidelines](CONTRIBUTING.md) first.
 
-This is the repository for awesome-semantic-shapes. You're welcome to contribute! Let's make the Web rock our socks
-off!
+Semantic shapes enables you to validate RDF graphs against a set of conditions. Semantic shapes can also be viewed as a description of the data graphs that do satisfy these conditions. Such descriptions may be used for a variety of purposes beside validation, including user interface building, code generation and data integration. Semantic shapes could be described in SHACL or ShEx languages.
```

Comment:
- "enable"
- use Semantic Newlines: https://sembr.org/

Reply:
Ugh. Semantic Newlines documents "the `git diﬀ` command" (note the ligature) which doesn't exist; it's the `git diff` command.

I do encourage more frequent line breaks, so lines are shorter and generally fit into a single window width. I find 80 chars or so is generally best for most devices. I find semantic newlines *less* readable, because they frequently deliver wider lines.

Reply:
Sorry for my stupidity, but where did you get `ﬀ`? My browser shows two separate `ff` in all commands on the https://sembr.org webpage, on the github page, in the raw README.md markdown.

Reply:
No stupidity in your question. Possibly some in programmers of Chrome or one of my active Chrome plugins, that's displaying ligatures without my express permission.

<img width="889" alt="image" src="https://github.com/user-attachments/assets/7c1b170e-8c1e-4ef4-8e7a-b837de9d28df">

Reply:
In mine it looks normal and I could select separate 'f' characters.
![image](https://github.com/user-attachments/assets/efb08fd0-f5d1-4843-b2cb-b8e48d6120b6)